Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -78,6 +78,11 @@ public function index(Document $document, $indexName)
                 case Field::TYPE_ARRAY:
                     $fields[$massiveField->getName()] = $value;
                     break;
+                case Field::TYPE_DATE:
```

Comment:
this code snippet is only a check for provided types. for string or array i wouldnt create a converter. the string converter would return a string and the array converter would have to decide if he returns a array or a string (for lucene) ... therefor i would refactor this check here and not implement converter for this two types!